scala_modules/.scalafmt.conf

```diff
@@ -0,0 +1,19 @@
+version = 2.0.0-RC5
```

Comment:
i assume this is standard or part of the twitter guidelines? comment would be good

Reply:
yeah, just needs to be pinned to something for the CLI check to work—added a comment